.circleci/config.yml

```diff
@@ -1,30 +1,59 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
 version: 2
-jobs:
-  build_node_v10:
-    docker:
-      - image: circleci/node:10
 
-    working_directory: ~/repo
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.11.3
```

Comment:
Is there a reason why the tests for this project and most on interledger are 8.11.3. Isn't the LTS 8.12.0? https://nodejs.org/en/